conjure-java-core/src/main/java/com/palantir/conjure/java/services/dialogue/DefaultStaticFactoryMethodGenerator.java

```diff
@@ -127,6 +142,45 @@ public MethodSpec generate(ServiceDefinition def) {
         return method;
     }
 
+    private MethodSpec createHelperToConstructExceptionDeserializerArgs(String serviceDefinitionPackageName) {
+        List<ErrorDefinition> errorDefinitions = packageToErrorDefinitionsMapping.get(serviceDefinitionPackageName);
```

Comment:
Mentioned in the tests PR, but I'm not sure we should limit ourselves to the errors from the same package as the service? (if anything, I could see devs adding the errors under a specific `.errors` package)